### PR TITLE
upated for new API

### PR DESCRIPTION
**Pull request by pperle**
New command "-l", "--location"
changed locale  from l to L

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -240,5 +246,30 @@ def clear_pokemon(self):
...
         print "Cleared " + str(cleared) + " names"
 
+    def get_elevation_for_position(self):
+        try:
+            url = 'https://maps.googleapis.com/maps/api/elevation/json?locations={},{}'.format(
+                str(self.position[0]), str(self.position[1]))
+            altitude = requests.get(url).json()[u'results'][0][u'elevation']
+            print "Local altitude is: {0}m".format(altitude)
+
+            self.position = (self.position[0], self.position[1], altitude)
```
Comment (tegob) [inline main.py:257]: Don’t forget to add a random value that is generated to the altitude returned by the Google Maps API. People don’t put their phones on the ground while playing Pokémon GO. I’d suggest something like `random.uniform(0.9, 1.7)` meters.

**Comment by tegob** [comment]
LGTM 👍

File: main.py [hunk 2]
```diff
@@ -31,10 +38,11 @@ def init_config(self):
         parser.add_argument("-a", "--auth_service")
         parser.add_argument("-u", "--username")
         parser.add_argument("-p", "--password")
+        parser.add_argument("-l", "--location")
```
Comment (tegob) [inline main.py:41]: I’d rather remove the shorthand than suddenly giving it a new meaning

Comment (pperle) [inline main.py:41]: location is a must, so a shorthand is always appreciated.
I set "l" for location and "L" for local as PokemonGo-Mapis using the same seetings [see](https://github.com/PokemonGoMap/PokemonGo-Map/blob/bc02e61aa5d5e6f1840955e1eb56fb490be4ac9b/pogom/utils.py#L47).

**Comment by chuparCh0pper** [comment]
Pul your location into here

`https://maps.googleapis.com/maps/api/elevation/json?locations={},{}`

to check what you actual altitude is

**Pull request merged**